misk-actions/src/main/kotlin/misk/web/interceptors/LogRequestResponse.kt

```diff
@@ -10,7 +10,20 @@ package misk.web.interceptors
  *
  * If arguments and responses may include sensitive information, it is expected that the toString()
  * methods of these objects will redact it.
+ *
+ * Rate limiting is used to sample the number of requests logged. The value specified is the
+ * buckets/sec allotted per service per pod. By default we set rate limiting for both successes
+ * and errors to 10 buckets/sec, enough to show things are happening without sending too many logs.
```

Comment:
Maybe let's start off more aggressive and see if people complain. Let's do 1 log event / sec